conda_setup.py

```diff
@@ -6,10 +6,9 @@
 from Cython.Build import cythonize
 
 extensions = [Extension('pyscipopt.scip', ['pyscipopt/scip.pyx'],
-                        extra_compile_args=['-UNDEBUG'],
```

Comment:
Sure. I thought that `UNDEBUG` would actually turn *off* the debug flags, but it seems that I'm wrong.